docs/stdlib/array.rst

```diff
@@ -141,14 +138,15 @@ Reference
         db> select [(x := 1, y := 1), (x := 2, y := 3.3)][1];
         {(x := 2, y := 3.3)}
 
-    Negative indexing is supported:
+    This operator also allows accessing elements through negation.
```

Comment:
Would be nice to briefly explain how this works for a user who might not know. Suggestion:
> This operator also allows accessing elements from the end of the array using a negative index.